src/DynamoCoreWpf/Views/Core/AnnotationView.xaml

```diff
@@ -383,7 +385,9 @@
                         </ListBox.Items>
                     </ListBox>
                 </MenuItem>
-                <MenuItem Name="ChangeSize" Header="{x:Static p:Resources.GroupContextMenuFont}">
```

Comment:
Should we remove either of these if no longer used?

Reply:
@QilongTang the ChangeSize menuitem is still used my change was only applying a Style to the MenuItem so we can have a fixed Width.